translations/stripes-core/en.json

```diff
@@ -131,5 +131,10 @@
 
   "createResetPassword.header": "Choose a password",
   "createResetPassword.newPassword": "New Password",
-  "createResetPassword.confirmPassword": "Confirm Password"
+  "createResetPassword.confirmPassword": "Confirm Password",
+
+  "routeErrorBoundary.sub": "An error occurred resulting in the current page becoming unstable.",
```

Comment:
What about something like, 

> An error occurred on this page and it could not be loaded.

I have heard from a few folks that the word "unstable" really freaks people out and conveys that the entire FOLIO system of frontend, backend, _OMG everything!_, is unstable. (I think many end-users don't necessarily distinguish a UI module from an Okapi module; it's all just "FOLIO" to them.) If this is officially-approved language, I don't want to mess with it, but if we're in charge, something less alarming would be great.

Reply:
Good point! I'll change it 👍